packages/workers-shared/utils/types.ts

```diff
@@ -0,0 +1,10 @@
+// import { z } from "zod";
+
+// export const RoutingConfigSchema = z.object({
+// 	hasUserWorker: z.boolean(),
+// });
+
+// export type RoutingConfig = z.infer<typeof RoutingConfigSchema>;
+export type RoutingConfig = {
+	hasUserWorker: boolean;
```

Comment:
I'm a bit confused. Is this the same routing config that is supposed to config things like `spaMode`/`trailingSlashes`/etc? Cause if so, adding `hasUserWorker` to it doesn't feel right to me. It's two very diff concerns

Reply:
Yep its the same routing config. Having the user Worker there does change the behaviour of Router Worker so it makes sense to me (whether to pass onto UW or return 404). It is _also_ used for other things, but ehhh I like reusing it more than creating another separate option. I think EWC is set up for it to be like that already 🫠